mlflow/java/src/main/java/org/mlflow/sagemaker/MLeapPredictor.java

```diff
@@ -1,18 +1,109 @@
 package org.mlflow.sagemaker;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import ml.combust.mleap.runtime.MleapContext;
+import ml.combust.mleap.runtime.frame.DefaultLeapFrame;
+import ml.combust.mleap.runtime.frame.Row;
 import ml.combust.mleap.runtime.frame.Transformer;
+import ml.combust.mleap.runtime.javadsl.BundleBuilder;
+import ml.combust.mleap.runtime.javadsl.ContextBuilder;
+import org.mlflow.utils.SerializationUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.collection.JavaConverters;
+import scala.collection.Seq;
 
 /** A {@link org.mlflow.sagemaker.Predictor} implementation for the MLeap model flavor */
 public class MLeapPredictor extends Predictor {
-  public MLeapPredictor(String modelDataPath, String inputSchemaPath) {}
+  private final Transformer pipelineTransformer;
+  private final LeapFrameSchema inputSchema;
+
+  private static final Seq<String> predictionColumnNames;
+  private static final Logger logger = LoggerFactory.getLogger(MLeapPredictor.class);
+
+  static {
+    List<String> predictionColumnList = Arrays.asList("prediction");
```

Comment:
This is hard-coded -- is it enforced somehow?

Reply:
I will remove this in favor of a string constant of the form `private static final String PREDICTION_COLUMN_NAME = "predictions"`

Reply:
Got it, let's document why this is a valid assumption (e.g., point to somewhere in docs or code).

Reply:
Made this change to a string constant and added informative documentation.